cmd/ci-secret-bootstrap/main.go

```diff
@@ -641,6 +641,29 @@ func mutateGlobalPullSecret(original, secret *coreapi.Secret) (bool, error) {
 		return false, fmt.Errorf("failed to marshal the docker config: %w", err)
 	}
 	original.Data[coreapi.DockerConfigJsonKey] = data
```

Comment:
This ends up getting overwritten below, I assume the goal is to append to it instead?

Reply:
yeah, that is the ideal scenario to append those secrets.